CMakeLists.txt

```diff
@@ -1971,6 +1971,7 @@ is_enabled(JNA HAS_JNA)
 is_enabled(ENABLE_ZERO_LENGTH_COORD_BOUND RELAX_COORD_BOUND)
 is_enabled(USE_CDF5 HAS_CDF5)
 is_enabled(ENABLE_ERANGE_FILL HAS_ERANGE_FILL)
+is_enabled(DISPATCH_VERSION 1)
```

Comment:
@WardF is this the correct way to do this in CMake?

Reply:
@edhartnett `is_enabled` is a macro created to map a 0/1, 'true/false' to an autotools-style 'yes/no'.  So in the example of `is_enabled(USE_CDF5 HAS_CDF5)`, `is_enabled` evaluates the value of `USE_CDF5` and assigns `HAS_CDF5` a value of either `yes` or `no`.  I believe these are then used when generating `nc-config` and `libnetcdf.settings`.

So, in short, I don't think `is_enabled(DISPATCH_VERSION 1)` does what you want it to do.

Reply:
...so what should I do instead? ;-)

Reply:
At an appropriate place in `CMakeLists.txt` (I would suggest near the top, where we define the netcdf version and SO version), add a `SET(NC_DISPATCH_VERSION 1)` statement.  This should propagate throughout when cmake generates files that reference it :).

Reply:
Will do...